deploy/template.yaml

```
@@ -120,6 +120,7 @@ Mappings:
       GOVUKNOTIFYTEMPLATEID: "3baf3b59-2396-44c8-b7a4-b5a981fad5e8"
       GOVUKNOTIFYDYNAMICEMAILTEMPLATEID: "fc1a7e2f-5186-4cb8-a8e0-81110b4e9e85"
       GOVUKNOTIFYFALLBACKMAILTEMPLATEID: "504a35f3-56f9-42f8-965c-cad58ff9b816"
+      GOVUKNOTIFYVCGENERATIONFAILUREEMAILTEMPLATEID: "890b5fa2-6693-41f4-a7ce-e7cce19ca32d"
```

I think we can skip the notify part to make the name shorter, will raise some tech debt to do the same for the rest

```suggestion
      VCGENERATIONFAILUREEMAILTEMPLATEID: "890b5fa2-6693-41f4-a7ce-e7cce19ca32d"
```